app/models/miq_report.rb

```diff
@@ -284,10 +284,10 @@ def col_format_hash
   end
 
   def format_row(row, allowed_columns = nil, expand_value_format = nil)
-    @tz ||= get_time_zone(Time.zone)
+    tz = User.current_user.get_timezone
```

Comment:
Some reports may have time profile attached to them and if user's Display timezone is default than ```get_time_zone``` should get timezone from report specific time profile. So, I would suggest 
```
tz = User.current_user&.settings.fetch_path(:display, :timezone) || get_time_zone(Time.zone)
```